x-pack/examples/ui_actions_enhanced_examples/public/drilldowns/dashboard_to_discover_drilldown/drilldown.tsx

```diff
@@ -47,9 +45,9 @@ export class DashboardToDiscoverDrilldown
     return [APPLY_FILTER_TRIGGER];
   }
 
-  private readonly ReactCollectConfig!: React.FC<CollectConfigProps>;
-
-  public readonly CollectConfig = reactToUiComponent(this.ReactCollectConfig);
+  public readonly CollectConfig = ((props: CollectConfigProps) => (
+    <CollectConfigContainer {...props} params={this.params} />
+  )) as any;
```

Comment:
is `any` necessary? seems like type works without it

Reply:
Removed.